src/slide.jsx

```diff
@@ -33,10 +36,12 @@ const Slide = React.createClass({
   setZoom() {
     const content = React.findDOMNode(this.refs.content);
     const zoom = (content.offsetWidth / config.width);
-    const contentScale = (content.parentNode.offsetHeight / config.height);
+    const contentScaleY = (content.parentNode.offsetHeight / config.height);
+    const contentScaleX = (content.parentNode.offsetWidth / config.width);
```

Comment:
agreed but there are some issues in the presenter mode and overview mode: the preview doesn't match the actual slide. Don't really know what to do. Need to dig in more